chia/full_node/mempool_manager.py

```diff
@@ -556,32 +509,32 @@ async def validate_spend_bundle(
             f"Cost: {cost} ({round(100.0 * cost/self.constants.MAX_BLOCK_COST_CLVM, 3)}% of max block cost)",
         )
 
-        return None, potential, list(conflicting_pool_items.keys())
+        return None, potential, [item.name for item in conflicts]
 
-    def check_removals(self, removals: Dict[bytes32, CoinRecord]) -> Tuple[Optional[Err], List[Coin]]:
+    def check_removals(self, removals: Dict[bytes32, CoinRecord]) -> Tuple[Optional[Err], Set[MempoolItem]]:
         """
         This function checks for double spends, unknown spends and conflicting transactions in mempool.
-        Returns Error (if any), dictionary of Unspents, list of coins with conflict errors (if any any).
+        Returns Error (if any), the sett of existing MempoolItems with conflicting spends (if any any).
```

Comment:
`Returns Error (if any), the set of existing mempool items with conflicting spends (if any).` ?